app/_includes/prereqs/auth/oidc/keycloak-password.md

```diff
@@ -0,0 +1,31 @@
+This tutorial requires an identity provider (IdP). If you don't have one, you can use [Keycloak](http://www.keycloak.org/). The steps will be similar in other standard identity providers.
+
+#### Create a client
+1. Install [Keycloak](https://www.keycloak.org/guides) (version 26 or later) on your platform.
+1. Open the admin console and select a realm. 
```

Comment:
```suggestion
1. Open the admin console
```
The one hang up I had was that I was looking for a realm before I realized that the keycloak master realm is pre selected